### ISSUE TITLE
tl-archerc7v2 build fails

### ISSUE THREAD

**Issue by xaque208** [freebsd/freebsd-wifi-build#73]
```

**Comment by xaque208** [comment]
Running `10.3-RELEASE`.

**Comment by erikarn** [comment]
Yeah, it doesn't handle running on stable/10; mkuzip / mkulzma got folded into the same tool on -head.

We'd have to patch the build infrastructure to handle it, make it easier to configure, or something. Sorry!

**Comment by xaque208** [comment]
Um, so how should I proceed?  Got any build infra coming up for this stuff?  Do I need to install HEAD in a jail or something and build there?

**Comment by erikarn** [comment]
Ugh, give me a day. I'll go figure it out for stable/10. It shouldn't be
that hard; i just have to bring some sanity to the code creating the
fsimage to make it easy to select mkulzma or mkuzip -L.


-adrian

**Comment by xaque208** [comment]
Its cool, no rush.  My wifi just really sucks and this device I have isn't doing anything at the moment.  I figure getting FreeBSD on it should help me deprecate my old wifi network.
